README.md

```diff
@@ -1,135 +1,104 @@
-# Camunda Platform 8 - Get Started
-
-This repository contains a short guide to get started with [Camunda
-Platform 8](https://camunda.com/products/cloud/). It contains instructions on how to
-model your first process, create a user task form, and automate a service task.
-During the guide you will use Console, Modeler, Zeebe, Operate, and Tasklist.
-
-# Table of contents
-
-- [Camunda Platform 8 - Get Started](#camunda-platform-8---get-started)
-- [Table of contents](#table-of-contents)
-- [Repository structure](#repository-structure)
-- [Set up the environment](#set-up-the-environment)
-- [Model the process](#model-the-process)
-  - [Model the user task form](#model-the-user-task-form)
-  - [Configure user task to use form](#configure-user-task-to-use-form)
-  - [Configure service task](#configure-service-task)
-- [Deploy process](#deploy-process)
-  - [Deploy using Camunda Modeler](#deploy-using-camunda-modeler)
-  - [Deploy using Web Modeler](#deploy-using-web-modeler)
-  - [Deploy using zbctl](#deploy-using-zbctl)
-  - [Deploy using code](#deploy-using-code)
-- [Start process instance](#start-process-instance)
-  - [Start instance using Camunda Modeler](#start-instance-using-camunda-modeler)
-  - [Start instance using Web Modeler](#start-instance-using-web-modeler)
-  - [Start instance using zbctl](#start-instance-using-zbctl)
-  - [Start instance using code](#start-instance-using-code)
-- [Complete the user task](#complete-the-user-task)
-- [Complete the service task](#complete-the-service-task)
-- [Further references](#further-references)
-  - [Developer setup of Camunda 8](#developer-setup)
+# Get started with Camunda 8
+
+This repository gets users started with [Camunda 8](https://camunda.com/platform/) and contains instructions on how to:
+
+- Model your first process
+- Create a user task form
+- Automate a service task
+
+During this guide, you will use Console, Modeler, Zeebe, Operate, and Tasklist.
```

Comment:
Would we want to add a note regarding Optimize? Something like "To leverage your process data and analyze areas for improvement, [get started with Optimize](https://docs.camunda.io/docs/next/guides/improve-processes-with-optimize/).

Reply:
I think that makes sense, could be a follow-up?